packages/migrations/src/batched-migrations/batched-migrations-runner.ts

```diff
@@ -279,7 +279,8 @@ export class BatchedMigrationsRunner extends EventEmitter {
 
     // Spin until we're no longer running.
     while (this.running) {
-      await sleep(1000);
+      // Long waits during shutdown cause the process to be terminated.
+      await sleep(5);
```

Comment:
I don't understand what this means. Why would a wait cause a process to be terminated? If there is indeed a problem here, a better solution is probably to unref the timer; see the third argument of this function.

Reply:
I added some info logging just before the `process.exit(0)` in `server.ts` and it was never reached. I changed the timeout and the exit logging was reached. My interpretation is that there is a process that sends a `SIGTERM`, waits 1 second and then sends `SIGKILL` (or `kill -9`). This is a common approach since it is impossible to tell the difference between a slow process and a hung process.

Reply:
What is this process that's sending a SIGKILL after a timeout? PL itself won't do that, and I'm not aware of any process in our Docker container that would do that either.